chapters/develop-organize/overview/reading/README.md

```diff
@@ -1,18 +1,50 @@
 ## Developing and Organizing Content
 
-Owner: Răzvan Nițu
-Reviewer: Răzvan Deaconescu
+Developing educational content is a time consuming activity that most educators would rather avoid.
+Typically, formal education institutions from similar fields have similar curriculums.
+As such, it would be much more time effective to share the educational materials used for one course rather than build them from scratch.
+To facilitate this, we propose a methodology for developing and organizing the educational materials such that the content is easily distributable, customizable, contributable and publishable.
+Ideally, once content has been created for a particular course or field of study, it will be usable by both institutions to teach the course or by self-taught individuals for self-study.
+In this chapter, we will present the main types of content that may be created to be used in educational activities.
+We discuss the strenghs and weakness for each of them and present guidelines and best practices which can be used for their creation.
+Finally, we present what are the best technological tools to create, present and maintain such resources.
 
-Present the structure and organization of a content repository.
-Detail its hierarchy (a `tree` command-like output) and summarize the types of contents in its subsections.
-
-The mapping between types of content and teaching activities is presented in the [Introduction](../../../../README.md).
-
-Sections:
+The topics presented in this chapter are:
 
 - [Reading](../../reading/reading/README.md)
 - [Media](../../media/reading/README.md)
 - [Slides](../../slides/reading/README.md)
 - [Guides](../../guides/reading/README.md)
 - [Drills](../../drills/reading/README.md)
 - [Projects](../../projects/reading/README.md)
+
+For the educational content to be available, we use a version control system (namely, `Git`) which makes it easy to store, use and propose changes to existing content.
```

Comment:
I would start with some general metrics / goals / properties of developed content and the methods for developing it:

- Being able to have a history of changes.
- Publicly available
- Ability to customize
- Ability to contribute
- Automate as many actions as possible
- Easy to write and update

You mentioned them above. And for each I would then detail the use of Markdown, Git, GitHub, linters, open source tools, open content tools (reveal.md  / draw.io), open source infrastructure. And reference the infrastructure for those. That's when they would be detailed, but it's good to tie different chapters well together.